ovn-tester/ovn_tester.py

```diff
@@ -184,6 +193,16 @@ def run_base_cluster_bringup(ovn, bringup_cfg):
                                            bringup_cfg.n_pods_per_node)
             worker.ping_ports(ovn, ports)
 
+def run_test_density(ovn, cfg):
+    context_name = "heavy_density" if cfg.lb else "light_density"
+    ns_address_set = ovn.nbctl.address_set_create('NameSpace')
```

Comment:
We keep all the "internals" in the ovn_workload module.

How about using the `Namespace` class?  Something like:
```
ns = Namespace(ovn, f'ns_{context_name}')
```
and below, after `provision_ports()`:
```
ns.add_port(ports[0])
```